browser-test/src/applicant/questions/northstar_id.test.ts

```diff
@@ -0,0 +1,262 @@
+import {Page} from '@playwright/test'
+import {test, expect} from '../../support/civiform_fixtures'
+import {
+  AdminPrograms,
+  AdminQuestions,
+  enableFeatureFlag,
+  loginAsAdmin,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+} from '../../support'
+
+test.describe('Id question for applicant flow', () => {
+  test.beforeEach(async ({page}) => {
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+  })
+
+  test.describe('single id question', () => {
+    const programName = 'Test program for single id'
+
+    test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
+      await setUpProgramWithSingleIdQuestion(
+        programName,
+        page,
+        adminQuestions,
+        adminPrograms,
+      )
+    })
+
+    test('validate screenshot with north star flag enabled', async ({
```

Comment:
**nitpick:** Seems like we could remove "with north star flag enabled" since the name of the file is `northstar_id.test.ts`.